coderd/database/dbauthz/querier.go

```diff
@@ -1068,7 +1068,11 @@ func (q *querier) UpdateUserHashedPassword(ctx context.Context, arg database.Upd
 
 	err = q.authorizeContext(ctx, rbac.ActionUpdate, user.UserDataRBACObject())
 	if err != nil {
-		return err
+		// Admins can update passwords for other users.
+		err = q.authorizeContext(ctx, rbac.ActionUpdate, user.RBACObject())
+		if err != nil {
+			return err
+		}
```

Comment:
Good catch; this should probably warrant a separate unit test

Reply:
It is here: https://github.com/coder/coder/blob/5b07f1e2a399b9acc568159fdac81b93552deda3/coderd/users_test.go#L617-L617

Reply:
Whoops, I missed it!